index.html

```diff
@@ -45,6 +45,133 @@ <h2 class="tx-center">Full-Stack Software Developer</h2>
+      <h5>WORK</h5>
+    </div>
+    <h3>MY RECENT<br>WORK</h3>
     <hr>
-
-</body>
\ No newline at end of file
+    <div class="g-container">
+      <img class="g-item1" src="img/Snapshoot-Portfolio.png" alt="Imagen de prueba">
+      <h4 class="g-item2">Multi-Post Stories</h4>
+      <p class="g-item3">
+        A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. has been the industrys standard 
+        dummy text ever since the 1500s, when an unknown printer took a 
+        standard dummy text.
+      </p>
+      <a class="g-item4">css</a>
+      <a class="g-item5">html</a>
+      <a class="g-item6">bootstrap</a>
+      <a class="g-item7">Ruby</a>
+      <button class="g-item8" type="button">See Project</button>
+      <div class="blur-elipse"></div>
+    </div>
+    <div class="g-container">
+      <img class="g-item1" src="img/Snapshoot-Portfolio.png" alt="Imagen de prueba">
+      <h4 class="g-item2">Multi-Post Stories</h4>
+      <p class="g-item3">
+        A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. has been the industrys standard 
+        dummy text ever since the 1500s, when an unknown printer took a 
+        standard dummy text.
+      </p>
+      <a class="g-item4">css</a>
+      <a class="g-item5">html</a>
+      <a class="g-item6">bootstrap</a>
+      <a class="g-item7">Ruby</a>
+      <button class="g-item8" type="button">See Project</button>
+      <div class="blur-elipse-2"></div>
+    </div>
+    <div class="g-container">
+      <img class="g-item1" src="img/Snapshoot-Portfolio.png" alt="Imagen de prueba">
+      <h4 class="g-item2">Multi-Post Stories</h4>
+      <p class="g-item3">
+        A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. has been the industrys standard 
+        dummy text ever since the 1500s, when an unknown printer took a 
+        standard dummy text.
+      </p>
+      <a class="g-item4">css</a>
+      <a class="g-item5">html</a>
+      <a class="g-item6">bootstrap</a>
+      <a class="g-item7">Ruby</a>
+      <button class="g-item8" type="button">See Project</button>
+      <div class="blur-elipse-3"></div>
+    </div>
+    <div class="g-container">
+      <img class="g-item1" src="img/Snapshoot-Portfolio.png" alt="Imagen de prueba">
+      <h4 class="g-item2">Multi-Post Stories</h4>
+      <p class="g-item3">
+        A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. has been the industrys standard 
+        dummy text ever since the 1500s, when an unknown printer took a 
+        standard dummy text.
+      </p>
+      <a class="g-item4">css</a>
+      <a class="g-item5">html</a>
+      <a class="g-item6">bootstrap</a>
+      <a class="g-item7">Ruby</a>
+      <button class="g-item8 mr-b" type="button">See Project</button>
+      <div class="blur-elipse-4"></div>
+    </div>
```

Comment:
- [ ] Vertical scroll does not create a good user experience. Please remove the vertical scroll from the page.

![image](https://user-images.githubusercontent.com/43995830/153714686-9318516a-b24b-43d1-9977-65c0ac26ad52.png)